models/job.go

```diff
@@ -106,7 +106,7 @@ type JobSpecNotifier struct {
 }
 
 type JobSpecTask struct {
-	Unit     *Plugin
+	Unit     *Plugin `json:"-" yaml:"-"`
```

Comment:
what scenarios this change will impact?

Reply:
Test file where we marshal and unmarshal specs.

Reply:
Alright